core/src/nucleus/ribosome/callback/validate_entry.rs

```diff
@@ -96,7 +96,7 @@ fn run_validation_callback(
     ) {
         Ok(call_result) => match call_result.is_null() {
             true => CallbackResult::Pass,
-            false => CallbackResult::Fail(call_result),
+            false => CallbackResult::Fail(call_result.to_string()),
```

Comment:
FYI `bool` is not a monad. Could you please make this a regular if-expression? 🙏

Reply:
@sphinxc0re i feel like i've touched enough LOC  already :P

was this way when i found it